adapters/IronSource/IronSourceAdapter/GADMAdapterIronSourceBannerAd.h

```diff
@@ -0,0 +1,30 @@
+//
+//  GADMAdapterIronSourceBannerAd.h
+//  ISMedAdapters
+//
+//  Created by alond on 11/12/2022.
+//  Copyright © 2022 ironSource Ltd. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import <GoogleMobileAds/GoogleMobileAds.h>
+#import <IronSource/IronSource.h>
+
+
+
+
+@interface GADMAdapterIronSourceBannerAd : NSObject 
+
+/// Initializes a new instance with |adConfiguration| and |completionHandler|.
+- (instancetype)initWithGADMediationBannerConfiguration:(GADMediationBannerAdConfiguration *)adConfiguration
+                                      completionHandler:(GADMediationBannerLoadCompletionHandler)
+                                                        completionHandler NS_DESIGNATED_INITIALIZER;
```

Comment:
Can you rename and update to below:
```
/// Asks the receiver to render the ad configuration.
- (void)loadBannerAdForAdConfiguration:(nonnull GADMediationBannerAdConfiguration *)adConfiguration
                     completionHandler:
                         (nonnull GADMediationBannerLoadCompletionHandler)completionHandler;
```

Also, can you clarify why do we need `NS_DESIGNATED_INITIALIZER`?